src/routes/posts.rs

```diff
@@ -227,7 +227,6 @@ pub fn update(blog: String, slug: String, user: User, cl: ContentLen, form: Leni
             post.license = form.license.clone();
             post.cover_id = form.cover;
             post.update(&*conn, &searcher).expect("post::update: update error");;
-            let post = post.update_ap_url(&*conn).expect("post::update: update ap url error");
```

Comment:
Just a note: I removed that without replacing it with something else, since it is incorrect (the AP IDs should change), and was not working anyway (since we were checking if `ap_url` was empty before updating it, which was only the case on newly created articles).